src/main/java/tr/ogu/edu/school/schoolapp/controller/ActTicketController.java

```diff
@@ -38,4 +40,11 @@ public ResponseEntity<Resource> getTicketAsPdf(@PathVariable String verification
 				.header(HttpHeaders.CONTENT_DISPOSITION, "inline;filename=" + verificationCode + ".pdf").body(resource);
 	}
 
-}
\ No newline at end of file
+	// Biletin kullanım durumu
+	@PostMapping("/verify")
```

Comment:
```suggestion
	@PostMapping("/public/{verificationCode}/{line}/{no}")
```
şeklinde gelecek bir isteği karşılamalı. Şu anda oluşturulan karekod bir URL adresi barındırıyor ve bu URL adresini GET ile açtığımızda üstteki metot(getTicketAsPdf) çalışıyor. Aynı adres POST ile çağrılacak. verify vs gibi bir gereksinim bulunmuyor.